opencti-platform/opencti-front/src/utils/hooks/useEntitySettings.tsx

```diff
@@ -73,17 +73,24 @@ export const useYupSchemaBuilder = (
   isCreation: boolean,
   exclusions?: string[],
 ): ObjectSchema<{ [p: string]: unknown }> => {
+  // simplest case: we're in update mode, so we do not need all mandatory fields
+  if (!isCreation) {
+    return Yup.object().shape(existingShape);
+  }
```

Comment:
This is what caused the initial problem described in the bug.

I'd really like a review on this, this is a rough change.
I assumed that this function had 2 modes, edition and creation, and we actually want to return a yup validation computed from schema ONLY on creation, because in update mode nothing is actually required, the query might patch any field.